.github/workflows/release.yml

```diff
@@ -139,38 +153,173 @@ jobs:
       - name: Bundle libusb DLL
         run: Copy-Item "C:\vcpkg\installed\x64-windows\bin\libusb-1.0.dll" -Destination "app\build\bin\"
 
+      - name: Create artifacts directory
+        run: New-Item -ItemType Directory -Force -Path artifacts
+
       - name: Create zip
         run: |
           cd app/build/bin
-          Compress-Archive -Path usb-tree.exe,libusb-1.0.dll -DestinationPath ../../../usb-tree-windows-amd64.zip
+          Compress-Archive -Path usb-tree.exe,libusb-1.0.dll -DestinationPath ../../../artifacts/usb-tree-windows-amd64.zip
```

Comment:
Create a task to bundle the icon with the .exe later on, and you should include the LICENSE in this .zip as well

Reply:
I imagine the `icon.ico` is supposed to accomplish that, did you test it?

Reply:
I don't see LICENSE being added